test/controllers/changeset_controller_test.rb

```diff
@@ -2036,37 +2046,46 @@ def test_list_max_id
   ##
   # This should display the last 20 changesets closed
   def test_feed
+    changeset = create(:changeset, :num_changes => 1)
+    _empty_changeset = create(:changeset, :num_changes => 0)
+
```

Comment:
Ah yes you're right... I misread the code in `check_feed_result` and thought it as filtering out open changesets.